pages/common/pyats-shell.md

```diff
@@ -0,0 +1,16 @@
+# pyats shell
+
+> Starts a pre-loaded pyATS interactive Python Shell to save time in prototyping.
+> More information: <https://pubhub.devnetcloud.com/media/genie-docs/docs/cli/genie_shell.html>.
+
+- Open pyATS shell with defined Testbed file:
+
+`pyats shell --testbed-file {{/path/to/your/testbed.yaml}}`
+
+- Open pyATS shell with defined Pickle file:
```

Comment:
```suggestion
- Open pyATS shell with a defined Pickle file:
```
Usually we don't capitalize in command descriptions but seems like the main documentation follows it too, so I guess it is fine.